index.js

```diff
@@ -153,9 +167,9 @@ app.get('/slackauth', (req, res) => {
 });
 
 // Starts Local server -- COMMENT OUT FOR DEPLOYMENT
-// app.listen(process.env.PORT || PORT, () => {
```

Comment:
can you put better instructions on whta needs to be changed for local dev? and are you checking the right things in?